src/main/java/org/hawkular/client/android/auth/AuthData.java

```diff
@@ -23,14 +23,12 @@
 public final class AuthData {
     public static final String NAME = "hawkular";
     public static final String STORE = "sessionStore";
-    public static final String DESCRIPTION = "{\"attributes\":{\"description\":\"" +
-            "'Token created via Hawkular Android Client'\"}}";
 
     public static final class Endpoints {
         private Endpoints() {
         }
 
-        public static final String ACCESS = "/secret-store/v1/tokens/create";
+        public static final String ACCESS = "/hawkular/metrics/tenants";
```

Comment:
When person ender username and password. I have to check if it is correct or not. Earlier it was we could check on the persona but as it is not there I found this uri as a good point to verify using HTTP code of response